proc_dash/utility.py

```diff
@@ -95,6 +104,12 @@ def count_unique_subjects(data: pd.DataFrame) -> int:
     )
 
 
+def count_unique_records(data: pd.DataFrame) -> int:
+    if all(col in data.columns for col in ["participant_id", "session"]):
```

Comment:
again, quite  :cherries:, but I find 

```suggestion
    if set(["participant_id", "session"]).issubset(data.columns):
```
a bit easier to read. (didn't test it)